src/helm/benchmark/server.py

```diff
@@ -12,6 +12,11 @@
 app = Bottle()
 
 
+@app.get("/using_release")
```

Comment:
The name of this could be improved (and probably the format of the data I return from the endpoint as well)

Reply:
This needs to be something like `/benchmark_output/using_release.json`

Also add a corresponding sample `using_release.json` file to the `static` directory. I believe the non-pattern route should override the pattern route, but you might need to check.

Reply:
Done.

Reply:
Also, verified that the pattern route is overriden by the explicit, non-pattern route.